### ISSUE TITLE
Composer errors

### ISSUE THREAD

**Issue by designvoid** [M165437/laravel-blueprint-docs#12]
Hi there,
Getting the following error when installing via composer? Not sure if it's related to your fork of drafter-php?

```
m165437/laravel-blueprint-docs v0.4.0 requires hmaus/drafter-php dev-symfony-v4 -> no matching package found.
```

**Comment by v-dyatlov** [comment]
Hi,
you can fix it by temporary adding github.com/M165437/drafter-php as VCS repo in your composer.json

**Comment by tisuchi** [comment]
Hi, same issue here. 

What is the status here right now? Still, I am facing this problem.

**Comment by alexgiuvara** [comment]
@M165437 I think you should publish your version of `drafter-php` on packagist.org. I tried to contact Hendrik without success.

**Comment by alexgiuvara** [comment]
I get this error: `/app/vendor/bin/drafter: not found`. My composer.json looks like:
```

**Comment by alexgiuvara** [comment]
Yes. It seems that I got an error when trying to install drafter:
```
libtool: unrecognized option `-static'
libtool: Try `libtool --help' for more information.
make[1]: *** [out/Release/libdrafter.a] Error 1
make: *** [drafter] Error 2
```
is not related to your package, sorry

**Comment by M165437** [comment]
Fixed with release [v0.4.1](https://github.com/M165437/laravel-blueprint-docs/tree/v0.4.1)
